lib/modules/manager/buildkite/extract.ts

```diff
@@ -5,7 +5,10 @@ import { GithubTagsDatasource } from '../../datasource/github-tags';
 import { isVersion } from '../../versioning/semver';
 import type { PackageDependency, PackageFileContent } from '../types';
 
-export function extractPackageFile(content: string): PackageFileContent | null {
+export function extractPackageFile(
+  content: string,
+  packageFile?: string
```

Comment:
```suggestion
  packageFile: string
```

Can it really be undefined? Or only in tests?

Reply:
yes, because of tests. making them not optional will cause a lot more changes in tests. i can fix that in follow up PR's.